packages/contracts-bedrock/contracts/test/L1Block.t.sol

```diff
@@ -41,16 +47,24 @@ contract L1BlockTest is CommonTest {
         uint256 fo,
         uint256 fs
     ) external {
```

Comment:
I think that we should not change this test. Its dedicated / focused solely on the setter for L1Blocks and individual getters for lastest bn. 

I think we should simply add the `assertEq(lb.getL1BlockHash(n)` and assert that it equals `h`